flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionCapacitySchedulerITCase.java

```diff
@@ -153,68 +153,62 @@ public void testTaskManagerFailure() {
 
 		// ------------------------ Test if JobManager web interface is accessible -------
 
-		YarnClient yc = null;
-		try {
-			yc = YarnClient.createYarnClient();
-			yc.init(YARN_CONFIGURATION);
-			yc.start();
+		final YarnClient yc = YarnClient.createYarnClient();
+		yc.init(YARN_CONFIGURATION);
+		yc.start();
 
-			List<ApplicationReport> apps = yc.getApplications(EnumSet.of(YarnApplicationState.RUNNING));
```

Comment:
These changes are not relevant to the issue at hand.

Reply:
True, but I came across while testing the Dispatcher REST endpoint in the YARN case. I think there is some value in letting the test fail with the proper stack trace instead of only failing the test with the failure message.